content/en/docs/components/pipelines/reference/kfp-kubernetes.md

```diff
@@ -0,0 +1,9 @@
++++
+title = "Kubernetes Platform-specific Features"
+description = "Reference documentation for the `kfp-kubernetes` Python library"
+weight = 6
+                    
++++
+
+The `kfp-kubernetes` Python library enables authoring Kubeflow pipelines with Kubernetes-specific features. These features are supported by the default KFP open source BE. 
+See the [reference documentation](https://kfp-kubernetes.readthedocs.io).
```

Comment:
let's change BE to backend, at the moment it's not clear

Reply:
I just removed the sentence.